lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetWriters.java

```diff
@@ -202,6 +212,9 @@ private static PrimitiveValueWriter getValueWriter(ValuesWriter valuesWriter, Ty
             verifyParquetType(type, parquetType, OriginalType.TIME_MICROS);
             return new TimeMicrosValueWriter(valuesWriter, parquetType);
         }
+        if (type instanceof TimestampType && parquetType.getPrimitiveTypeName().equals(PrimitiveType.PrimitiveTypeName.INT96)) {
+            return new Int96TimestampValueWriter(valuesWriter, type, parquetType, parquetTimeZone);
+        }
         if (TIMESTAMP_MILLIS.equals(type)) {
             verifyParquetType(type, parquetType, OriginalType.TIMESTAMP_MILLIS);
             // TODO when writing with Hive connector, isAdjustedToUTC is being set to true, which might be incorrect
```

Comment:
is this TODO still relevant?

Reply:
The TODO looks out of date with the code as isAdjustedToUTC was already false, I got rid of it now